lightning/src/chain/keysinterface.rs

```diff
@@ -265,30 +277,29 @@ pub trait BaseSign {
 	/// This is required in order for the signer to make sure that the state has moved
 	/// forward and it is safe to sign the next counterparty commitment.
 	fn validate_counterparty_revocation(&self, idx: u64, secret: &SecretKey) -> Result<(), ()>;
-
-	/// Create a signatures for a holder's commitment transaction and its claiming HTLC transactions.
-	/// This will only ever be called with a non-revoked commitment_tx.  This will be called with the
-	/// latest commitment_tx when we initiate a force-close.
-	/// This will be called with the previous latest, just to get claiming HTLC signatures, if we are
-	/// reacting to a ChannelMonitor replica that decided to broadcast before it had been updated to
-	/// the latest.
+	/// Creates a signature for a holder's commitment transaction and its claiming HTLC transactions.
+	/// This will only ever be called with a non-revoked `commitment_tx`. This will be called with the
+	/// latest `commitment_tx` when we initiate a force-close.
+	/// This will be called with the previous latest `commitment_tx`, just to get claiming HTLC
+	/// signatures, if we are reacting to a [`ChannelMonitor`] replica that decided to broadcast before
```

Comment:
Monitor replica is obscure for an outsider without explanation here: https://github.com/lightningdevkit/rust-lightning/blob/main/GLOSSARY.md#monitor-replicas

Reply:
Added a link to the glossary (even though I'm not sure it provides much more clarity in its current form).